.github/workflows/scripts/e2e-verify.sh

```diff
@@ -19,13 +19,16 @@ source "./.github/workflows/scripts/e2e-utils.sh"
 
 BRANCH="main"
 
+# Note: this requires GH_TOKEN to be set in the workflows.
+THIS_FILE=$(gh api -H "Accept: application/vnd.github.v3+json" /repos/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID | jq -r '.path' | cut -d '/' -f3)
```

Comment:
Is in necessary to add the Accept header? Is it not set by default? What does the gh command do without it?

Reply:
I don't think it's set by default because the GH documentation adds it in their example. I just copied the commands from the docs. But it's possible that it it is set by default and the doc is out of date